inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -849,6 +866,15 @@ private ReificationStrategy getReificationStrategy(KnowledgeBase kb)
             return new NoReification(this);
         }
     }
+    
+    // Method to create and define base property
```

Comment:
Comment should be JavaDoc comment and also explain what a "base property" and when this method should be used instead of `createProperty`.

Reply:
Java doc comment is there in KBService.